app/assets/js/rtMedia.js

```diff
@@ -547,6 +547,33 @@ jQuery( 'document' ).ready( function( $ ) {
 
 	function rtmedia_init_popup_navigation() {
 		var rtm_mfp = jQuery.magnificPopup.instance;
+		
+		var probablyMobile = rtm_mfp.probablyMobile;
+		var tooltipShown   = getCookie( 'rtmedia-touch-swipe-tooltip' );
+
+		// Check if its mobile and tooltip is first time dispaly.
+		if ( probablyMobile && "" === tooltipShown ) {
+
+		    // Show tooltip.
+		    jQuery( '#mobile-swipe-overlay' ).show();
+		    
+		    // On touch hide tooltip.
+		    jQuery( '#mobile-swipe-overlay' ).on ( 'click', function( e ) {
+			setCookie( 'rtmedia-touch-swipe-tooltip' , true, 365 );
+			jQuery( this ).hide();
+		    } );
+		    
+		    // On swipe hide tooltip.
+		    jQuery( '#mobile-swipe-overlay' ).swipe( {
+			 //Generic swipe handler for all directions
+			swipe:function(event, direction, distance, duration, fingerCount, fingerData) {
```

Comment:
space before and after `(`

could be `swipe:function( event, direction, distance, duration, fingerCount, fingerData )`